src/utils.ts

```diff
@@ -68,3 +68,17 @@ export function getGatewayUserAgent(): string {
 
   return `${primary} ${secondary}${tertiary}`;
 }
+
+export function isWebthingsThingDescription(thingDescription: {
+  '@context': string | string[];
+}): boolean {
+  if (typeof thingDescription['@context'] === 'string') {
+    return thingDescription['@context'] === 'https://webthings.io/schemas';
+  }
+
+  if (Array.isArray(thingDescription['@context'])) {
+    return thingDescription['@context'].includes('https://webthings.io/schemas');
+  }
+
+  return false;
+}
```

Comment:
This unfortunately doesn't work because the [`@context`](https://webthings.io/api/#web-thing-description) member in a Web Thing Description is optional. It's only needed if a developer wants to add semantic annotations using schemas from the schema repository at https://webthings.io/schemas/. There are also some adapters (like the virtual things adapter) which still set `@context` to https://iot.mozilla.org/schemas/

I think this function needs to work the other way around and check whether a Thing Description is a W3C Thing Description by looking for a `@context` of 

1. https://www.w3.org/2019/wot/td/v1
2. http://www.w3.org/ns/td
3. Whatever the 1.1 context is going to be

 as per https://github.com/w3c/wot-scripting-api/blob/main/typescript/thing-description/schema/td-json-schema-validation.json#L63